dawn/renderer/utils/utils.js

```diff
@@ -41,11 +41,17 @@ export const robotState = {
 // TODO: Synchronize this and the above state
 export const runtimeState = {
   STUDENT_CRASHED: 0,
+  0: 'Crashed',
   STUDENT_RUNNING: 1,
+  1: 'Running',
   STUDENT_STOPPED: 2,
+  2: 'Stopped',
   TELEOP: 3,
+  3: 'Tele-Operated',
   AUTONOMOUS: 4,
+  4: 'Autonomous',
   ESTOP: 5,
+  5: 'E-Stop',
```

Comment:
Just for the record, this will likely never get printed except momentairly. After receiving e-stop, runtime kills itself (this is deliberate)